### PR TITLE
Turkey: Updated admin2 name translations

### LINKED ISSUE #1527: Update English preferred names in Turkey admin2 records
https://spelunker.whosonfirst.org/id/85632393/descendants/?exclude=nullisland&placetype=county

County records in Turkey often have parenthetical names appended to the end of the English preferred name. In these cases, we should swap the English preferred and variant names.

Examples:
- https://spelunker.whosonfirst.org/id/890462793/
  - "Adalar (Istanbul)" vs. "Adalar"
- https://spelunker.whosonfirst.org/id/890462627/
  - "Kadiköy (Istanbul)" vs. "Kadiköy"

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #1527.

- Moves preferred English names that contain a parenthetical to a variant name property.
- Strips the parenthetical off of the existing English preferred name.

These are property changes to admin2 in Turkey and do not require PIP work. Can merge as-is.

### CODE REVIEW

File: data/110/870/930/9/1108709309.geojson [hunk 1]
```diff
@@ -44,10 +44,11 @@
         "Avc\u0131lar"
     ],
     "name:eng_x_preferred":[
-        "Avcilar (Istanbul)"
+        "Avcilar"
     ],
     "name:eng_x_variant":[
-        "Avc\u0131lar"
+        "Avc\u0131lar",
+        "Avcilar (Istanbul)"
```
Comment (nvkelso) [inline data/110/870/930/9/1108709309.geojson]: Nit: These should also be stored into the disambiguation property?

Comment (stepps00) [reply data/110/870/930/9/1108709309.geojson]: Good call. I am working through a few property edit issues right now, but will circle back to add them in this branch. That'll be a nice first use case.

Comment (stepps00) [reply data/110/870/930/9/1108709309.geojson]: Added via acf37e5.

**Comment by stepps00** [comment]
@nvkelso - okay to merge with the most recent commits?

**Pull request merged**